src/shell/scripts/omp.ps1

```diff
@@ -35,8 +35,7 @@ New-Module -Name "oh-my-posh-core" -ScriptBlock {
         )
 
         if ($ExecutionContext.SessionState.LanguageMode -eq "ConstrainedLanguage") {
-            $_arguments = $Arguments -join ' '
-            $standardOut = Invoke-Expression "& $FileName $_arguments 2>&1"
+            $standardOut = Invoke-Expression "& '$FileName' `$Arguments 2>&1"
```

Comment:
@JanDeDobbeleer 🤦🏻 I'm sorry that I missed a case where `$FileName` may contain single quotes, which will mess it up. We have to change this line to:

```powershell
$standardOut = Invoke-Expression "& `$FileName `$Arguments 2>&1"
```